lib/commands.js

```diff
@@ -637,7 +637,7 @@ exports.bindCommands = function bindCommands(forum) {
             args.shift();
             commandText = args.shift();
             mention = true;
-        } else if (/^!\S{3,}(\s|$)/.test(line)) {
+        } else if (/^!\w\S{2,}(\s|$)/.test(line)) {
```

Comment:
I'm not too keen on this change myself. `\w` allows the underscore character and disallows anything that is not ASCII alphanumeric, so the command: `!去跳舞` (Simplified Chinese: Go Dance) will be rejected by the updated code where it would be accepted by the current test.

Is this a change we want to make at this time? wouldn't it be better to leave the existing test and and a "and not match" for `/!\[\S+\]\(\S+\)(\s|$)/` to remove the one case we want to stop matching?